contrib/runners/inquirer/tests/unit/test_inquirer_runner.py

```
@@ -0,0 +1,116 @@
+
+runner_params = {
+    "users": [],
+    "roles": [],
+    "tag": "developers",
+    "schema": {}
+}
+
+
+class InquiryTestCase(RunnerTestCase):
+
+    def test_runner_creation(self):
+        runner = inquirer.get_runner()
+        self.assertTrue(runner is not None, 'Creation failed. No instance.')
+        self.assertEqual(type(runner), inquirer.Inquirer, 'Creation failed. No instance.')
+
+    @mock.patch('inquirer.TriggerDispatcher', mock_trigger_dispatcher)
+    @mock.patch('inquirer.action_utils', mock_action_utils)
+    @mock.patch('inquirer.action_service', mock_action_service)
+    def test_simple_inquiry(self):
+        runner = inquirer.get_runner()
+        runner.context = {
+            'user': test_user
+        }
+        runner.action = self._get_mock_action_obj()
+        runner.runner_parameters = runner_params
+        runner.container_service = service.RunnerContainerService()
+        runner.pre_run()
+        mock_liveaction_db.context = {
+            "parent": test_parent_id
+        }
+        (status, output, _) = runner.run({})
+        self.assertEqual(status, LIVEACTION_STATUS_PENDING)
+        self.assertTrue(output is not None)
+        self.assertEqual(output, {"response_data": {}})
+        mock_trigger_dispatcher.return_value.dispatch.assert_called_once_with(
+            'core.st2.generic.inquiry',
+            {
+                'users': [],
+                'roles': [],
+                'liveaction_id': None,
+                'tag': "developers",
+                'response': {},
+                'schema': {}
+            }
+        )
+        mock_action_service.request_pause.assert_called_once_with(
+            test_parent_id,
+            test_user
+        )
+
+    @mock.patch('inquirer.TriggerDispatcher', mock_trigger_dispatcher)
+    @mock.patch('inquirer.action_utils', mock_action_utils)
+    @mock.patch('inquirer.action_service', mock_action_service)
+    def test_inquiry_failed_no_parent(self):
```

Hrm, correct me if I'm remembering it incorrectly, but I thought we decided that while it was **possible** to run these actions standalone, we should throw an error in that case. To facilitate testing, maybe we should have an optional parameter you can pass that causes the runner to ignore this check? WDYT?